proxies/live/apiproxy/targets/scr-target.xml

```diff
@@ -24,9 +24,17 @@
       <Step>
         <Name>OauthV2.VerifyAccessToken</Name>
       </Step>
+      <Step>
+        <Name>AssignMessage.SetAccessModeApplicationRestricted</Name>
+      </Step>
+      <Step>
+        <Name>AssignMessage.SetAccessModeUserRestricted</Name>
+        <Condition>(scope JavaRegex "(.+\ urn:nhsd:apim:user-nhs-id:aal3:summary-care-record\ .+|^urn:nhsd:apim:user-nhs-id:aal3:summary-care-record\ .+|.+\ urn:nhsd:apim:user-nhs-id:aal3:summary-care-record$|^urn:nhsd:apim:user-nhs-id:aal3:summary-care-record$|.+\ summary-care-record:USER-RESTRICTED\ .+|^summary-care-record:USER-RESTRICTED\ .+|.+\ summary-care-record:USER-RESTRICTED$|^summary-care-record:USER-RESTRICTED$)")</Condition>
```

Comment:
can we look at reducing this to `<Condition>(scope JavaRegex "(.+\ summary-care-record:USER-RESTRICTED\ .+|^summary-care-record:USER-RESTRICTED\ .+|.+\ summary-care-record:USER-RESTRICTED$|^summary-care-record:USER-RESTRICTED$)")</Condition>` ? we changed it because it was not working, now we can reduce again!!